packages/legacy/core/App/screens/ProofRequest.tsx

```diff
@@ -407,25 +404,19 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
           <>
             <ConnectionImage connectionId={proof?.connectionId} />
             <View style={styles.headerTextContainer}>
-              {!hasAvailableCredentials() || !hasSatisfiedPredicates(getCredentialsFields()) ? (
+              {!hasSatisfiedPredicates(getCredentialsFields()) ? (
                 <View style={{ flexDirection: 'row', alignItems: 'center' }}>
                   <Icon
                     style={{ marginLeft: -2, marginRight: 10 }}
                     name="highlight-off"
                     color={ListItems.proofIcon.color}
                     size={ListItems.proofIcon.fontSize}
                   />
-                  {hasSatisfiedPredicates(getCredentialsFields()) ? (
-                    <Text style={styles.headerText} testID={testIdWithKey('HeaderText')}>
-                      <Text style={[TextTheme.title]}>{proofConnectionLabel || t('ContactDetails.AContact')}</Text>{' '}
-                      {t('ProofRequest.IsRequestingSomethingYouDontHaveAvailable')}
-                    </Text>
-                  ) : (
-                    <Text style={styles.headerText} testID={testIdWithKey('HeaderText')}>
-                      {t('ProofRequest.YouDoNotHaveDataPredicate')}{' '}
-                      <Text style={[TextTheme.title]}>{proofConnectionLabel || t('ContactDetails.AContact')}</Text>
-                    </Text>
-                  )}
+
+                  <Text style={styles.headerText} testID={testIdWithKey('HeaderText')}>
+                    {t('ProofRequest.YouDoNotHaveDataPredicate')}{' '}
```

Comment:
Just checking `{' '}` is needed here. Some times my VSCode inserts stuff like this when formatting.

Reply:
not really sure, I didn't edit those lines I just updated the code around them so they show up as a change